flow/genericactors.actor.h

```diff
@@ -360,25 +360,25 @@ Future<Void> mapAsync( FutureStream<T> input, F actorFunc, PromiseStream<U> outp
 
 //Waits for a future to be ready, and then applies a function to it.
 ACTOR template<class T, class F>
-Future<typename std::result_of<F(T)>::type> map(Future<T> what, F func)
+Future<std::invoke_result_t<F, T>> map(Future<T> what, F func)
 {
 	T val = wait(what);
 	return func(val);
 }
 
 //maps a vector of futures
 template<class T, class F>
-std::vector<Future<typename std::result_of<F(T)>>> map(std::vector<Future<T>> const& what, F const& func)
+std::vector<Future<std::invoke_result<F, T>>> map(std::vector<Future<T>> const& what, F const& func)
```

Comment:
There are a few places that do `invoke_result_t`, and a few that do `invoke_result`. Is this an accident?

Reply:
Good catch! They should all be `invoke_result_t`.